ui/next.config.js

```diff
@@ -1,4 +1,5 @@
 module.exports = {
+  basePath: '/ui',
```

Comment:
Won't this break when running behind a proxy that serves the app under a path?

Reply:
I think so, we should probably have an issue to address the dynamic base path same way we (will) do in Admin UI (future because it's still not merged).

Reply:
Yes, and we don't need the base path config here after all. Removed it just now.